client/src/components/TagWorkspaceRole.vue

```diff
@@ -0,0 +1,41 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <ion-label class="label-role">
+    <ion-chip
+      class="caption-caption tag"
+      :class="workspacerole"
+    >
+      {{ translateWorkspaceRole(workspacerole) }}
+    </ion-chip>
+  </ion-label>
+</template>
+
+<script setup lang="ts">
+import { IonChip, IonLabel } from '@ionic/vue';
+import { defineProps } from 'vue';
+import { WorkspaceRole } from '@/common/mocks';
+import { useI18n } from 'vue-i18n';
+
+const { t } = useI18n();
+
+defineProps<{
+  workspacerole: WorkspaceRole,
```

Comment:
Should be `workspaceRole` or just `role`.

Reply:
I'm updating it.